src/ucs/debug/memtrack.h

```diff
@@ -28,6 +29,15 @@ void ucs_memtrack_allocated(void *ptr, size_t size, const char *name);
  */
 void ucs_memtrack_releasing(void *ptr);
 
+/**
+ * Allocates a path buffer of size PATH_MAX.
+ *
+ * @param buffer Pointer to the allocated memory.
+ * @param name   Name for logging.
+ * @return UCS_OK on success, UCS_ERR_NO_MEMORY on failure.
+ */
+ucs_status_t ucs_alloc_path_buffer(char **buffer, const char *name);
```

Comment:
IMO this file (`memtrack`) is not the place for it..

Reply:
maybe `string.h`?

Reply:
Yes

Reply:
Moved to `string.h`

Reply:
Not pushed

Reply:
please check now